lib/sqlalchemy/orm/session.py

```diff
@@ -1904,6 +1905,10 @@ def begin_nested(self) -> SessionTransaction:
             with the SQLite driver in order for SAVEPOINT to work
             correctly.
 
+            :ref:`aiosqlite_serializable` - special workarounds required
```

Comment:
**Federico Caselli** (CaselIT) wrote:

as the other comment

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4907

Reply:
Fixed